app/src/androidTest/java/com/github/bgabriel998/softwaredevproject/points/GeonamesHandlerTest.java

```diff
@@ -251,7 +256,8 @@ public void testGeonamesObjCreationExceptionQueeryMaxAsZero() throws IllegalArgu
 
         Context context = ApplicationProvider.getApplicationContext();
         Assert.assertNotNull(context);
-        UserPoint userPoint = new UserPoint(46.519251915333676,6.558563221333525,220);
+        UserPoint userPoint = UserPoint.getInstance(context);
+        userPoint.setLocation(46.519251915333676, 6.558563221333525, 220, 0);
```

Comment:
Use constants instead
```suggestion
        userPoint.setLocation(MOCK_LOCATION_LAT_LAUSANNE, MOCK_LOCATION_LON_LAUSANNE, MOCK_LOCATION_ALT_LAUSANNE, 0);
```